src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/command/BroadcastWrapper.java

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.command;
+
+import org.openkilda.model.SwitchId;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+import java.util.Set;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class BroadcastWrapper extends CommandData {
+    @JsonProperty("scope")
+    private Set<SwitchId> scope;
```

Comment:
why not switches? scope is too abstract:-p

Reply:
"switches" is even more abstract here as for me. It doesn't bring any "restriction" of "affected area" meaning only the type of data stored in the set.

"switch" is most used term in our system, it is the main focus in open-kilda, so this term is everywhere. If there is any possibility to not use the term "switch", I will prefer to not use it.